src/pattern-library/components/patterns/prototype/SelectNextPage.tsx

```diff
@@ -216,11 +222,31 @@ export default function SelectNextPage() {
 
           <Library.Example title="Disabled Select">
             <Library.Demo title="Disabled Select">
-              <div className="w-[350px] mx-auto">
+              <div className="w-96 mx-auto">
                 <SelectExample disabled />
               </div>
             </Library.Demo>
           </Library.Example>
+
+          <Library.Example title="Select with long content">
+            <p>
+              <code>SelectNext</code> makes sure the content never overflows,
```

Comment:
Are you referring to the button or listbox content here?

Reply:
I think I wrote this at one point when I had a different approach in mind than what I ended up pushing 😅 

I'll re-write it to make it more clear.